docs/sources/articles/systemd.md

```diff
@@ -31,7 +31,7 @@ If the `docker.service` file is set to use an `EnvironmentFile`
 referenced file.
 
 Or, you may need to edit the `docker.service` file, which can be in `/usr/lib/systemd/system`
-or `/etc/systemd/service`.
+or `/etc/systemd/service` or `/lib/systemd/system`.
```

Comment:
ya, and then the oxford comma for the final `or`

with oxford-or-correctness fix, LGTM

@docker/docs-owners